api_schema/schema.go

```diff
@@ -524,6 +524,17 @@ type PostClientPacketCaptureStatsRequest struct {
 	// Report on any errors encounted during apidump that should be shown to the user.
 	ApidumpError     ApidumpErrorType `json:"apidump_error,omitempty"`
 	ApidumpErrorText string           `json:"apidump_error_text,omitempty"`
+
+	// Report CPU and memory usage, if available.
+	AgentUsage *AgentUsage `json:"agent_usage,omitempty"`
```

Comment:
`AgentResourceUsage` and `agent_resource_usage` would be more descriptive.

Reply:
Changed in 9a50cbd4827ff573e2d59bc69a8cfe7f73971278.